benchmark848/benchmark.cpp

```diff
@@ -0,0 +1,348 @@
+    "CREATE NODE TABLE Customer (id INT32, firstName STRING, lastName STRING, company STRING, city STRING, country STRING, primaryPhone STRING, secondaryPhone STRING, email STRING, website STRING, PRIMARY KEY(id));",
+    "CREATE NODE TABLE Organization (id INT32, name STRING, website STRING, country STRING, description STRING, foundYear INT16, industry STRING, numEmployee INT16, PRIMARY KEY(id));"
+};
+/* will be updated later */
+string tableCopyQuery [NUM_TABLES] = {"", "", ""};
+
+const vector<string> tableColumns[NUM_TABLES] = {{"id", "firstName", "lastName", "sex", "email", "phone", "jobTitle"},
+    {"id", "firstName", "lastName", "company", "city", "country", "primaryPhone", "secondaryPhone", "email", "website"},
+    {"id", "name", "website", "country", "description", "foundYear", "industry", "numEmployee"}};
+const int tableNumColumns[3] = {7, 10, 8};
+
+enum AlterType : int
+{
+    DROP_COLUMN = 0,
+    ADD_COLUMN = 1
+};
+
+enum AlterAddColumnType : int
+{
+    INVALID_COLUMN_TYPE = -1,
+    INT64 = 0,
+    BOOL = 1,
+    STRING = 2
+};
+
+enum TestType : int
+{
+    DROP_TABLE = 0,
+    ALTER_TABLE = 1,
+    DELETE_NODE_GROUP = 2
+};
+
+/* Helper functions */
+std::string GenRandomStr(const int len) {
+    static const char alphanum[] =
+        "0123456789"
+        "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
+        "abcdefghijklmnopqrstuvwxyz";
+    std::string tmp_s;
+    tmp_s.reserve(len);
+
+    for (int i = 0; i < len; ++i) {
+        tmp_s += alphanum[rand() % (sizeof(alphanum) - 1)];
+    }
+    
+    return tmp_s;
+}
+
+char* GetCmdOption(char ** begin, char ** end, const std::string & option)
+{
+    char ** itr = std::find(begin, end, option);
+    if (itr != end && ++itr != end)
+    {
+        return *itr;
+    }
+    return nullptr;
+}
+
+void UpdateTableCopyQuery(string csvFileDir)
+{
+    tableCopyQuery[0] = "COPY People FROM '" + csvFileDir + +"/people-10000.csv';";
+    tableCopyQuery[1] = "COPY Customer FROM '" + csvFileDir + +"/customers-10000.csv';";
+    tableCopyQuery[2] = "COPY Organization FROM '" + csvFileDir + +"/organizations-10000.csv';";
+}
+
+string RandomSelectTable(string avoidTable)
+{
+    string tableName = avoidTable;
+    while (tableName == avoidTable) {
+        int tableIndex = rand() % NUM_TABLES;
+        tableName = tableNames[tableIndex];
+    }
+
+    return tableName;
+}
+
+int FindTableIndex(string tableName)
+{
+    for (int i = 0; i < NUM_TABLES; i++) {
+        if (tableNames[i] == tableName) {
+            return i;
+        }
+    }
+
+    KU_ASSERT(0);
+    return 0;
+}
+
+void CreateTable(const unique_ptr<Connection> &connection, string tableName)
+{
+    for (auto i = 0; i < NUM_TABLES; i++) {
+        auto name = tableNames[i];
+        if (tableName == name) {
+            connection->query(tableCreateQuery[i]);
+            connection->query(tableCopyQuery[i]);
+        }
+    }
```

Comment:
Minor - it seems we can directly use the `findTableIndex` function here:
```
int idx = findTableIndex(tableName);
connection->query(tableCreateQuery[idx]);
connection->query(tableCopyQuery[idx]);
```

Reply:
Fixed in the newest commit.